lib/web_catalog/api_extractor.es6.js

```diff
@@ -678,10 +678,14 @@ foam.CLASS({
 
       const type = og.getType(nameId);
       return !(this.constantTypes.includes(type) &&
-               (og.lookupMetaData(name, id).value === 1 ||
-                // If type is boolean, value of false causes "value" to be 0;
-                // return true ignoring "value" metadata.
-                type === 'boolean'));
+                  (og.lookupMetaData(name, id).value === 1 ||
+                      // If type is boolean, value of false causes "value" to be
+                      // 0; return true ignoring "value" metadata.
+                      type === 'boolean') &&
+                  this.isConstName_(name));
+    },
+    function isConstName_(name) {
+      return /^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$/.test(name);
```

Comment:
Can you link to https://heycam.github.io/webidl/#idl-grammar and say this is the all-caps subset of that? I think it doesn't matter too much if it's not an exact subset, leading underscore and dashes in the name is, I think, a hypothetical.

Could also simplify the expression somewhat to `/^[A-Z][A-Z0-9_]*$/` to allow for double underscores. Or comment here why *not* allowing double underscores is important.